source/MissionAction.cpp

```diff
@@ -33,6 +33,15 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 using namespace std;
 
 namespace {
+	void DoGift(PlayerInfo &player, const Ship *model, const string &name, UI *ui)
+	{
+		if(model->ModelName().empty())
+			return;
+		
+		player.BuyShip(model, name, true);
+		Messages::Add("The " + model->ModelName() + " \"" + name + ",\" was added to your fleet.");
```

Comment:
Either a comma both before & after the name, or no comma at all

Reply:
```suggestion
		Messages::Add("The " + model->ModelName() + " \"" + name + "\" was added to your fleet.");
```
Okay, whatever.